models/user/user.go

```diff
@@ -346,7 +346,6 @@ func GetUserFollowing(ctx context.Context, u, viewer *User, listOptions db.ListO
 		Select("`user`.*").
 		Join("LEFT", "follow", "`user`.id=follow.follow_id").
 		Where("follow.user_id=?", u.ID).
-		And("`user`.type=?", UserTypeIndividual).
```

Comment:
could you change this condition to be `type in (0, 1)` as we now have more types than just users and orgs?